packages/atlas/src/views/viewer/ChannelView/ChannelView.styles.ts

```diff
@@ -166,7 +166,7 @@ export const TabsContainer = styled.div<TabsContainerProps>`
   background-color: #000;
 
   ${media.sm} {
-    align-items: center;
+    align-items: flex-start;
```

Comment:
After discussion with @toiletgranny we decided to align items inside `TabsContainer` to the top.